### PR TITLE
RAC-5253  support on-web-ui CI/CD

### PR DESCRIPTION
**Pull request by changev**
Add on-web-ui into CI/CD process.

https://github.com/RackHD/on-web-ui/pull/88
https://github.com/RackHD/RackHD/pull/836

@panpan0000 @PengTian0

### CODE REVIEW

**Review by HQebupt — APPROVED** [review]

File: vmslave-config/ansible/group_vars/all [hunk 1]
```diff
@@ -2,7 +2,8 @@
 sources_list_url: http://**.**.**.***/static/misc/sources.list
 
 #Versions
-node_version: 4.7.3
+node_4_version: 4.7.3
+node_6_version: 6.10.0
```
Comment (PengTian0) [inline vmslave-config/ansible/group_vars/all:6]: Why install 2 versions of node?

**Comment by PengTian0** [comment]
on-web-ui is not enabled in FunctionTest in this PR, is that expected?

**Comment by changev** [comment]
@PengTian0 Yes, on-web-ui doesn't have any function test scripts.

File: vmslave-config/ansible/group_vars/all (hunk 1 above)
Comment (changev) [reply vmslave-config/ansible/group_vars/all:6]: build on-web-ui depends on node6

**Comment by panpan0000** [comment]
-1
 please show test result and the brief of the steps

**Pull request closed without merging**